src/pages/admin/user/index.jsx

```diff
@@ -0,0 +1,136 @@
+import { Box, Container } from '@mui/material';
+import React, { useState } from 'react';
+import {
+  NavBarDashBoard, SearchDashboard, SideBar, Table,
+} from '../../../components';
+import FormDashBoard from '../../../components/formDashboard/FormDashBoard';
+import UserContext from '../../../context';
+
+const columns = [

```

Comment:
these two arrays should be separated in a file in the same folder. name it dummyData.js

Reply:
done